openfe/tests/data/181l.pdb

```diff
@@ -0,0 +1,1840 @@
+HEADER    HYDROLASE (O-GLYCOSYL)                  19-APR-95   181L              
```

Comment:
Also - for future considerations, is there a way we can parse compressed PDB files? It might be quite a useful thing when folks are trying to use OpenFE at scale.

Reply:
> Also - for future considerations, is there a way we can parse compressed PDB files?

[Isn't this all we need?](https://docs.python.org/3/library/zipfile.html) (obviously also with something to read PDB, but we already have that)

Reply:
I _think_ so, seems like the rdkit examples just use `gzip` but either should work hopefully?

Reply:
[Standard library has zip, gzip, bz2, tar, and more!](https://docs.python.org/3/library/archiving.html)